e2e/pages/Dashboard.js

```diff
@@ -113,6 +106,7 @@ class Dashboard extends Page {
      * @param workspaceName for the new workspace
      * @returns {Workspaces} return an instance of workspace
      */
+    //TODO move to new class - workspace.js
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)